### PR TITLE
Enable intl en-US and pt-BR

### PR DESCRIPTION
**Pull request by ChristoferDantas**
PR to handle #3 

I enabled the localization feature of the winforms, and I've done a strategy to click in the FLAG in order to change the language of the app.

### CODE REVIEW

**Comment by markiing** [comment]
Hey @ChristoferDantas . First of all, thanks for your contribution.

I've noticed some points.

1. The application restart too when i click in current language (If i'm in PT-BR and click in Brazilian flag, the application shouldn't restart)

2. We could change cursor to "pointer" instead of "cross" in flags.

**Comment by ChristoferDantas** [comment]
@markiing changes made

File: Program.cs [hunk 1]
```diff
@@ -9,6 +12,11 @@ internal static class Program
         [STAThread]
         static void Main()
         {
+            /// Set default localization
+            var language = ConfigurationManager.AppSettings["language"];
+            Thread.CurrentThread.CurrentCulture = new System.Globalization.CultureInfo(language);
+            Thread.CurrentThread.CurrentUICulture = new System.Globalization.CultureInfo(language);
```
Comment (afonsocdaniel) [inline Program.cs:18]:
```suggestion
```

**Comment by afonsocdaniel** [comment]
@ChristoferDantas  I would suggest to change from Icon(EN e PT) to a Combo box, so we can easily add more languages without to change the UI. Combo box allow us to scale when we add more translations

Just a non block comment:
In the future, we can move all the language file from an XML to a dictionary, txt or yaml so every user can easily add more languages without to edit the code, just open a PR adding a new file with the translation. With that approach, we can create a folder "languages" with all the translation and every user(can be a non developer), can easily add new translations.

**Comment by afonsocdaniel** [comment]
@ChristoferDantas I greatly appreciate your PR, thank you for the contribution.

**Comment by ChristoferDantas** [comment]
@afonsocdaniel I found a way to have a resource file for each language, but we would need to set the text of each element "manually" (1 line per item). Do you think it's a way to go?

**Comment by afonsocdaniel** [comment]
@ChristoferDantas don't worry about it right now, we have some idea to sure, maybe we can talk in call this week. What do you think?

For now, just add a combo box so we can merge this PR

**Comment by markiing** [comment]
> @afonsocdaniel I found a way to have a resource file for each language, but we would need to set the text of each element "manually" (1 line per item). Do you think it's a way to go?

Maybe this is a way @ChristoferDantas . UIs replacements can be made when Form is rendered. 
This way it's easier to add more languages and the application behaviour won't change.

What do you think?

**Pull request closed without merging**